lib/chef/provider/mount/aix.rb

```diff
@@ -149,6 +180,18 @@ def enable_fs
           end
         end
 
+        def mount_options_unchanged?
+          current_resource_options = @current_resource.options
+          current_resource_options.pop
```

Comment:
The concern is that if something changed, and we start popping the wrong thing, it's going to be hard for someone else walking into this code to understand why it is there.